IntegrationTests/common/script/Makefile.mk

```
@@ -42,8 +42,11 @@ MemPrints:
 	 elif [[ $${BASE_DIR} == "CombinedBarrelConfig" ]]; \
 	 then \
 	   ln -sf $(EMDATA)/MemPrintsCMBarrel MemPrints; \
+	 elif [[ $${BASE_DIR} == "CombinedConfig_FPGA2" ]]; \
+	 then \
+	   ln -sf $(EMDATA)/MemPrintsSplit MemPrints; \
 	 else \
-	   ln -sf $(EMDATA)/MemPrintsCM MemPrints; \
+	   ln -sf $(EMDATA)/MemPrints MemPrints; \
```

@aryd I don't think this line should change, since emData/MemPrints doesn't exist anymore.
```suggestion
	   ln -sf $(EMDATA)/MemPrintsCM MemPrints; \
```

---

Not sure why this was changed. I have undone this update now.